docs/REST API/rest-api.md

```diff
@@ -1872,6 +1872,7 @@ the `software` table.
 | os_id                   | integer | query | The ID of the operating system to filter hosts by.                                                                                                                                                                                                                                                                                          |
 | os_name                 | string  | query | The name of the operating system to filter hosts by. `os_version` must also be specified with `os_name`                                                                                                                                                                                                                                     |
 | os_version              | string  | query | The version of the operating system to filter hosts by. `os_name` must also be specified with `os_version`                                                                                                                                                                                                                                  |
+| cve                     | string  | query | The cve to filter hosts by (including "cve-" prefix, case-insensitive).                                                                                                                                                                                                                                                                     |
```

Comment:
```suggestion
| vulnerability                     | string  | query | The cve to filter hosts by (including "cve-" prefix, case-insensitive).                                                                                                                                                                                                                                                                     |
```

Maybe `vulnerabilities` is more consistent w/ the current API? 

Looks like we use `vulnerabilities` in `GET /hosts/:id`, `GET /software/titles`, and other endpoints: https://fleetdm.com/docs/rest-api/rest-api#default-response128

Rachael, I think up to you. Just wanted to raise this.